lib/memery.rb

```diff
@@ -31,6 +31,13 @@ def memoize(method_name, condition: nil, ttl: nil)
       define_memoized_method!(method_name, condition: condition, ttl: ttl)
     end
 
+    def memoized?(method_name)
+      return false unless defined?(@_memery_module)
+
+      @_memery_module.method_defined?(method_name) ||
```

Comment:
What if method is protected? I think it’s better to use Memery.method_visibility in this case.

Reply:
> What if method is protected?

`true`: https://ruby-doc.org/core-2.6.4/Module.html#method-i-method_defined-3F

> Public and protected methods are matched.